src/axom/primal/operators/evaluate_integral.hpp

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*!
+ * \file evaluate_integral.hpp
+ *
+ * \brief Consists of methods that evaluate integrals over regions defined
+ * by Bezier curves, such as 2D area integrals and scalar/vector field line integrals
+ *
+ * Line integrals are computed with 1D quadrature rules supplied
+ * by MFEM. 2D area integrals computed with "Spectral Mesh-Free Quadrature for Planar 
+ * Regions Bounded by Rational Parametric Curves" by David Gunderman et al.
+ */
```

Comment:
I appreciate this whole comment!  I see you wrote "with ... rules supplied by MFEM," as is accurate.  Please add the additional note, to make things extra clear:

```suggestion
 * Regions Bounded by Rational Parametric Curves" by David Gunderman et al.
 *
 * \note This requires the MFEM third-party library.
 */
```